app/javascript/mastodon/components/admin/ReportReasonSelector.jsx

```diff
@@ -8,6 +8,7 @@ import classNames from 'classnames';
 import api from 'mastodon/api';
 
 const messages = defineMessages({
+  legal: { id: 'report.categories.legal', defaultMessage: 'Legal' },
```

Comment:
Can you add a comment here explaining where those values come from, and a comment in the Ruby file where they are defined to remind people to update those?